public/app/features/variables/adhoc/AdHocVariableEditor.test.tsx

```diff
@@ -38,12 +38,7 @@ jest.mock('@grafana/runtime/src/services/dataSourceSrv', () => {
 });
 
 const props = {
-  extended: {
-    dataSources: [
-      { text: 'Prometheus', value: null }, // default datasource
-      { text: 'Loki', value: { type: 'loki-ds', uid: 'abc' } },
-    ],
-  },
+  extended: {},
```

Comment:
From what I can tell including `dataSources` in `props.extended` was always invalid, and has just been removed.

This is not a runtime change (it's just a test!) and the test still passes.